tests/wdio.conf.js

```diff
@@ -78,7 +79,7 @@ const baseConfig = {
     browserName: 'chrome',
     acceptInsecureCerts: true,
     'goog:chromeOptions': {
-      args: ['--headless', '--disable-gpu', '--deny-permission-prompts']
+      args: [/*'--headless',*/ '--disable-gpu', '--deny-permission-prompts']
```

Comment:
just curious, why are we commenting the `--headless`?

Reply:
both changes in `wdio.conf.js` are temporary, I'm trying to reproduce a bug in the CI that I have locally, sorry for the disruption